README.md

```diff
@@ -439,6 +439,29 @@ $ docker run -d -it -p 7200:8000 -p 7201:8001 -p 7202:8002 \
 
 When you complete these steps, you will have multiple containers; one on each VM and all connected to each other on the 'ml-cluster-network' network. All the containers will be part of same cluster.
 
+# Upgrading from Dcoker Release 10.0-1 or later
+
+To upgrade MarkLogic Docker from release 10.0-1 to the current release, perform following steps:
```

Comment:
I think we should mention the fact that this can work because we can reuse an existing volume and we can easily locate it if named volume was used with --mount.

Reply:
line 440 - comma after on each VM,

Reply:
> I think we should mention the fact that this can work because we can reuse an existing volume and we can easily locate it if named volume was used with --mount.

Step 2 in the upgrade process suggests using the same volume mounted to the container using an older version.

Reply:
> line 440 - comma after on each VM,

This line is not part of story CLD-387. I will record this and put it in a different story for GA readme corrections.